### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by UShan89** [spdx/spdx-online-tools#1]
@rtgdk I find that there are two requirements file and two src directories. 
I need your help with understanding how the repo is structured. 

Another tiny request (Personal thoughts): Please do a pip freeze and modify the requirements.txt for `jpype1` and `Django-downloadview`

i.e. lock down to specific versions of the libraries. This will help avoid bugs/deprecations with future versions. Each time, If there be a need to use a higher version, we can bump up the requirements.txt and have a new release generated. Wanted to hear to your thoughts.

**Comment by rtgdk** [comment]
@UShan89  Thanks, I have updated the `requirements.txt` to have specific versions.
Since the online tool uses java tools jar file for its working, we would always need the updated version of the java tool inside the online tool. So we can use `git submodule` for maintaining both the repo without any interference. The current size of the whole repo is about 184 MB with tools taking 138 MB. So I thought it would be better to have a jar inside the src directory for now ( `tool.jar` ) and make use of that jar for the time being, In this case, for testing we don't need to download the whole repo and can only download the `src` directory. That's why I kept `requirements.txt` in both the folder and the sub-folder.
